app/assets/javascripts/spree/frontend/spree_payu_integration.js

```diff
@@ -1 +1,31 @@
 /* Placeholder for backend dummy app */
+SpreePayuIntegration = {
```

Comment:
I'm not against the name at all just to not pollute global namespace. :) Like putting whole code inside an inline called function. like:

```
(function() {
  var SpreePayuIntegration = {....};
  // ... 
})();
```